src/js/view/controls/templates/nextup.ts

```diff
@@ -1,9 +1,13 @@
 import type { HTMLTemplateString } from 'types/generic.type';
+import { Browser } from 'environment/environment';
 
 export default (header = '', title = '', duration = '', closeAriaLabel = ''): HTMLTemplateString => {
+    // When using firefox this adds opacity to stop the firefox PiP button getting clicked instead of the next up div
+    const firefoxPipFix = Browser.firefox ? ' jw-nextup-firefox-pip-fix' : '';
```

Comment:
You can move this logic to line 32 of `nextuptooltip.js` to avoid adding logic to the template itself to keep things more SRP. It's a nit, so don't consider this a required change.